analyzer/build.gradle.kts

```diff
@@ -29,18 +29,37 @@ dependencies {
     shadow(localGroovy())
     shadow(gradleApi())
 
-    compile("commons-beanutils:commons-beanutils:1.9.3")
     compile("org.commonjava.maven.ext:pom-manipulation-core:${project.extra.get("pmeVersion")}")
-    testCompile("junit:junit:4.12")
-    testCompile("com.github.stefanbirkner:system-rules:1.19.0")
-    testCompile("org.jboss.byteman:byteman:${project.extra.get("bytemanVersion")}")
+    compile("org.commonjava.maven.ext:pom-manipulation-io:${project.extra.get("pmeVersion")}")
+    compile("org.commonjava.maven.ext:pom-manipulation-common:${project.extra.get("pmeVersion")}")
+    compile("org.commonjava.maven.atlas:atlas-identities:${project.extra.get("atlasVersion")}")
+
+    compile("org.apache.maven:maven-settings-builder:${project.extra.get("mavenVersion")}")
+    compile("org.apache.maven:maven-settings:${project.extra.get("mavenVersion")}")
+
+    compile("commons-lang:commons-lang:${project.extra.get("commonsVersion")}")
+    compile("commons-io:commons-io:${project.extra.get("commonsVersion")}")
+    compile("commons-codec:commons-codec:1.11")
+
+    compile("org.slf4j:slf4j-api:${project.extra.get("slf4jVersion")}")
+    compile("org.aeonbits.owner:owner-java8:${project.extra.get("ownerVersion")}")
+
+    testCompile("junit:junit:${project.extra.get("junitVersion")}")
+    testCompile("com.github.stefanbirkner:system-rules:${project.extra.get("systemRulesVersion")}")
+    testCompile("org.assertj:assertj-core:${project.extra.get("assertjVersion")}")
     testCompile("org.jboss.byteman:byteman-bmunit:${project.extra.get("bytemanVersion")}")
-    testCompile("org.jboss.byteman:byteman-submit:${project.extra.get("bytemanVersion")}")
-    testCompile("org.jboss.byteman:byteman-install:${project.extra.get("bytemanVersion")}")
     testCompile (files ("${System.getProperty("java.home")}/../lib/tools.jar") )
-    testCompile("org.assertj:assertj-core:3.12.2")
     testCompile("org.mockito:mockito-core:2.27.0")
     testCompile("com.github.tomakehurst:wiremock-jre8:2.23.2")
+
+    // Ignore our own modules; avoid https://github.com/wfhartford/gradle-dependency-analyze/issues/83
+    permitUsedUndeclared(project(":common"))
+    // Groovy is built into Gradle
+    permitUsedUndeclared("org.codehaus.groovy:groovy:2.5.7")
```

Comment:
Not sure I understand this one, why is it needed?

Reply:
Do you mean the `permitUsedUndeclared` /  `permitUsedUndeclared` or specifically for groovy ?

Reply:
Specifically for groovy.

Reply:
It reports groovy as `permitUsedUndeclared` but groovy is supplied by gradle so we don't need to declare it

Reply:
Interesting stuff, I had never seen that before

Reply:
It's from this plugin https://github.com/wfhartford/gradle-dependency-analyze

Reply:
:+1: